docs/faq.md

```diff
@@ -51,6 +51,29 @@ following:
 }
 ```
 
+The same technique works for extracting element attributes into variables.
+Here's an example of storing an element's text attribute.
+
+```swift
+// Swift
+/*
+ *  Example Usage:
+ *
+ *  var textValue:String = ""
+ *
+ *  domainField.performAction(grey_replaceText("hello.there"))
+ *             .performAction(grey_getText(&textValue))
+ */
+func grey_getText(inout text: String) -> GREYActionBlock {
+    return GREYActionBlock.actionWithName("get text",
+                                           constraints: grey_respondsToSelector(Selector("text")),
+                                          performBlock: { element, errorOrNil -> Bool in
```

Comment:
I'm not actually sure how to format this correctly. I made another attempt. Is there a Swift style guide?